pkgs/tools/networking/simple-dlna-browser/default.nix

```diff
@@ -0,0 +1,54 @@
+{ pkgs, lib, makeWrapper, ...}:
+let
+  buildInputs = with pkgs; [
+    gnused
+    socat
+  ];
+in
+pkgs.stdenv.mkDerivation {
+  pname = "simple-dlna-browser";
+  version = "unstable-2023-01-09";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "javier-lopez";
+    repo = "learn";
+    sparseCheckout = [
+      "sh/tools/simple-dlna-browser"
+    ];
+    rev = "666e2c402723251ba19ce9d7c11b83b06c15a2c5";
+    sha256 = "sha256-jwPAEgti5DrjNnNVW/61GGTP+QedGAjz9kPwYhMAdTk=";
+  };
+
+  inherit (buildInputs);
```

Comment:
Are those packages used at build time?

Reply:
`sed` is used in the install phase and `socat` is necessary for the script's full functionality.

Reply:
Found a few more dependencies in the script and added them to the args. Nothing is really built, so I could remove the line entirely in 268feff